### DISCUSSION TITLE
How to setup development environment for Allure2.

### DISCUSSION THREAD

**Discussion by snowblacksh** [allure-framework/allure2#1313] [General Discussion]
Hi ,

We need customize the allure report according to the project requirement. However, we don't find any official document about how to deploy the development environment, only got some information from https://stackoverflow.com/.
Is there any document about this topic provided? or List some major steps to help up setup this environment.  

Thanks
Amanda

**Comment by Avaddon-tfa** [comment]
Hey there! And sorry for staying silent for almost a month.

At the moment, we don't actually get what "development environment" means.
Usually, Allure is built by Gradle locally or at the CI.

Could you please provide more details and describe the exact task you are solving?
if we had the specific task, we'd be able to provide a specific guide to do it.

**Comment by CraigRichards** [comment]
I'm new to Java myself and found running gradlew solved most of my local build issues.

Hope this helps
